<launch>

	<arg name="tracker_name" default="cob_body_tracker" />
	<arg name="camera_name" default="camera" />

	<!--include file="$(find openni2_launch)/launch/openni2.launch">
		<arg name="depth_registration" value="true"/>
	</include-->

	<!-- Nodelet manager >
	<node pkg="nodelet" type="nodelet" ns="$(arg camera_name)" name="$(arg camera_name)_nodelet_manager" args="manager" output="screen">
		<param name="num_worker_threads" value="4" />
	<node-->



	<node pkg="nodelet" type="nodelet" name="body_tracker" args="load cob_openni2_tracker/body_tracker_nodelet /camera/camera_nodelet_manager" output="screen" cwd="ROS_HOME">
		<param name="camera_frame_id" type="str" value="$(arg camera_name)_link" /> <!-- $(arg camera_name)_depth_optical_frame -->
		<param name="tf_prefix" value="$(arg tracker_name)" />
		<param name="relative_frame" value="$(arg camera_name)_depth_optical_frame" />
	
		<rosparam file="$(find cob_openni2_tracker)/launch/body_tracker_params.yaml"/>
	
	</node>

</launch>
